src/Faker/Provider/id_ID/Address.php

```diff
@@ -85,15 +85,15 @@ class Address extends \Faker\Provider\Address
 
     /**
      * Abbreviated State Names.
-     * Currently this is all just a guesswork, but should be highly accurate
+     *
```

Comment:
How about adding a reference (@<!-- -->link) here?

Reply:
Thanks! 
I just cross-referenced the list with the one from Wikipedia - there's Kaltara missing in in stateAbbr and I can't find Kalut in Wikipedia. 
That's probably two names for the same province? Still, the list should definitely be identical to the one in the provided link, so could you please either find a link where Kaltara is replaced with Kalut, or fix it in the list?

Reply:
Done. It's hard to find a single page that contains the acronyms of all the states in Indonesian language. The acronyms on wiki page are accurate and, IMHO, can be used as a reference.

Reply:
Yes, you're right. The abbreviation of Kalimantan Utara is Kaltara.

Reply:
Thanks!